applications/rag/README.md

```diff
@@ -77,7 +65,7 @@ This filter can auto fetch the templates in your project. Please refer to the fo
     1. Inspect templates: https://cloud.google.com/dlp/docs/creating-templates-inspect
     2. De-identification templates: https://cloud.google.com/dlp/docs/creating-templates-deid
 
-#### Verify Setup
+#### Verify setup
```

Comment:
Why change the capitalization? No preference, just curious

Reply:
I also don't have a strong preference, but this is how we format metadata headers internally at Google.